Docs/developer_guide/build_instructions/linux.md

```diff
@@ -35,30 +35,35 @@ part of the *superbuild*:
   - Private
 - libXt
 
-### Debian 10 Stable (Buster)
+### Debian 11 Stable (Bullseye) and Testing (Bookworm) 
 
 Install the development tools and the support libraries:
 
 ```console
 sudo apt update && sudo apt install git subversion build-essential cmake cmake-curses-gui cmake-qt-gui \
-  qt5-default qtmultimedia5-dev qttools5-dev libqt5xmlpatterns5-dev libqt5svg5-dev qtwebengine5-dev qtscript5-dev \
+  qtmultimedia5-dev qttools5-dev libqt5xmlpatterns5-dev libqt5svg5-dev qtwebengine5-dev qtscript5-dev \
   qtbase5-private-dev libqt5x11extras5-dev libxt-dev libssl-dev
 ```
 
-### Debian Testing (Bullseye) and Debian 9
-
-*This option is not suggested since it does not work with standard packages.  Debian 9 Qt 5.7 packages will not work with current Slicer 4.11.  Checked 2020-08-19.  May be possible to build from source or install other packages.  In addition, for Debian 9 you also need to build cmake from source as [described here](https://cmake.org/install/) or otherwise get a newer version than is supplied by the distribution.*
+### Ubuntu 21.10 (Impish Indri)
 
 Install the development tools and the support libraries:
 
 ```console
 sudo apt update && sudo apt install git subversion build-essential cmake cmake-curses-gui cmake-qt-gui \
-  qt5-default qtmultimedia5-dev qttools5-dev libqt5xmlpatterns5-dev libqt5svg5-dev qtwebengine5-dev qtscript5-dev \
+  qtmultimedia5-dev qttools5-dev libqt5xmlpatterns5-dev libqt5svg5-dev qtwebengine5-dev qtscript5-dev \
   qtbase5-private-dev libqt5x11extras5-dev libxt-dev libssl-dev
 ```
 
 ### Ubuntu 20.04 (Focal Fossa)
 
+:::{note} Warning
+:class: warning
+Since the default Qt5 packages available on Ubuntu 20.04 correspond to version 5.12.8 and version 5.15.2 is used to build and test the packages available for download. Compiling Slicer against version 5.12.8 may not succeed, and if it does, the compiled Slicer application may behave differently.
+
+To use Qt 5.15.2, we recommend you download and install following these instructions.
```

Comment:
To use Qt 5.15.2, we recommend you download and install following [these instructions](#any-distribution).

Reply:
Done